src/views/product-integrations-landing/components/searchable-integrations-list/helpers/get-filtered-integrations.ts

```diff
@@ -0,0 +1,23 @@
+import { Integration } from 'lib/integrations-api-client/integration'
+
+const getFilteredIntegrations = ({
+	integrations,
+	filterQuery,
+}: {
+	integrations: Integration[]
+	filterQuery: string
+}) => {
+	return integrations.filter((integration: Integration) => {
+		return (
+			integration.name.toLowerCase().includes(filterQuery.toLowerCase()) ||
+			integration.description
+				.toLowerCase()
+				.includes(filterQuery.toLowerCase()) ||
+			integration.organization.slug
+				.toLowerCase()
+				.includes(filterQuery.toLowerCase())
```

Comment:
💭 ⛏️  I see this was copied over from what already was there in the file below. But just thinking outloud, there could be opportunity to DRY this up a bit to make it easier to read. Maybe we could start by extracting the lowercased filter query into a constant. Or making a function that runs the `includes` logic. Understandable if you just want to refactor into the function for now.

Reply:
Since this abstraction affects prod and there's already a lot of changes in this PR, I think it'd be good to refactor this in a separate PR where the scope can be focused and tested more easily!